### ISSUE TITLE
Warning: Unexpected input(s) 'bump-policy'

### ISSUE THREAD

**Issue by yonatanaftali** [phips28/gh-action-bump-version#155]
Using the bump-policy option, I'm getting the following error during run-time:
`Warning: Unexpected input(s) 'bump-policy', valid inputs are ['tag-prefix', 'minor-wording', 'major-wording', 'patch-wording', 'rc-wording', 'skip-tag', 'PACKAGEJSON_DIR', 'target-branch', 'default', 'preid', 'commit-message', 'push']`

This is my configuration:
```
      - name: Bump Version
        uses: phips28/gh-action-bump-version@v9.0.1
        with:
          minor-wording: MinorBump
          major-wording: MajorBump
          preid: alpha
          skip-tag: true
          commit-message: 'bump version => {{version}}'
          target-branch: preAlphas/preAlpha-${{ steps.today.outputs.date }}
          bump-policy: 'ignore'
```

**Comment by phips28** [comment]
I can only repeat the error message 😄  there is no bump-policy ?! where did you get this from?
_valid inputs are ['tag-prefix', 'minor-wording', 'major-wording', 'patch-wording', 'rc-wording', 'skip-tag', 'PACKAGEJSON_DIR', 'target-branch', 'default', 'preid', 'commit-message', 'push']_

**Comment by yonatanaftali** [comment]
@phips28 
![image](https://user-images.githubusercontent.com/57040227/158596323-3d8fae8a-ae86-485b-b20f-a22d4cc20c30.png)

**Comment by phips28** [comment]
I already got your error message...

As said: there is no bump-policy! where did you get this from?

**Comment by yonatanaftali** [comment]
@phips28 From your own documentation:
![image](https://user-images.githubusercontent.com/57040227/158597075-ccbd9f4c-a53a-4387-b0fc-05c153b0439c.png)

**Comment by dallen4** [comment]
@yonatanaftali thanks for letting me know, I'll submit a PR later today to fix that. @phips28 it was from my feature addition.

**Comment by phips28** [comment]
Oh I totally missed that this was added 🙈  sorry

**Comment by dallen4** [comment]
All good, I will take a look later today unless you already know how to handle it

**Comment by phips28** [comment]
No I dont know how to handle this 🙈

**Comment by dallen4** [comment]
@yonatanaftali so upon investigation, it looks like its a version issue. According to your code snippet, you're implementing `phips28/gh-action-bump-version@v9.0.1` but my `bump-policy` wasn't published until [v9.0.14](https://github.com/phips28/gh-action-bump-version/commit/031f5ec2b3d025d1ce2a4ccded50e4da2a482930). Current version is `v9.0.16` so I'd recommend changing your action to implement `phips28/gh-action-bump-version@v9` or `phips28/gh-action-bump-version@master`.

@phips28 don't believe any fix is necessary 🙂

**Comment by yonatanaftali** [comment]
@dallen4 @phips28 Ah, got it! So maybe you should update the marketplace page accordingly:
![image](https://user-images.githubusercontent.com/57040227/158649329-d8317dcc-a6ab-42a5-904e-a279d8b37fc0.png)
